### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jerajeuf**
Added positive commentary about SCRF to About Us section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 The [Smart Contract Research Forum](https://www.smartcontractresearch.org/) (SCRF) serves as an open, agnostic bridge between academia and industry. Supported by an international community dedicated to advancing actionable blockchain research, our mission is to surface challenging problems in the space and present them to an active community of academics, researchers, and developers. It is the ecosystem’s source for problems to solve, and to find the people, projects, and tools to solve them. 
 
+The SCRF mission is pretty neat! 🔥🔥🔥
```
Comment (loicdrbx) [inline README.md]: Why, thank you! We have a strict one emoji per line rule, so please modify this sentence accordingly.

**Review by loicdrbx — CHANGES_REQUESTED** [review]
Looks good. I've left a few comments that should be addressed before this gets merged.

File: README.md (hunk 1 above)
Comment (jerajeuf) [reply README.md]: kk. Will do.

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 The [Smart Contract Research Forum](https://www.smartcontractresearch.org/) (SCRF) serves as an open, agnostic bridge between academia and industry. Supported by an international community dedicated to advancing actionable blockchain research, our mission is to surface challenging problems in the space and present them to an active community of academics, researchers, and developers. It is the ecosystem’s source for problems to solve, and to find the people, projects, and tools to solve them. 
 
-The SCRF mission is pretty neat! 🔥🔥🔥
+The SCRF mission is pretty neat! 🔥
```
Comment (loicdrbx) [inline README.md:10]: Good job

**Review by loicdrbx — APPROVED** [review]

**Pull request merged**